.ci/scripts/generate-cucumber-reports.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+## Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+## or more contributor license agreements. Licensed under the Elastic License;
+## you may not use this file except in compliance with the Elastic License.
+
+set -euxo pipefail
+
+## original image: ystia/cucumber-html-reporter:latest
+readonly DOCKER_IMAGE="docker.elastic.co/observability-ci/cucumber-html-reporter:latest"
+
+CUCUMBER_REPORTS_PATH="${CUCUMBER_REPORTS_PATH:-""}"
+ARCHITECTURE="${ARCHITECTURE:-""}"
+PLATFORM="${PLATFORM:-""}"
+SUITE="${SUITE:-""}"
+TAGS="${TAGS:-""}"
+
+main() {
+  for f in "${CUCUMBER_REPORTS_PATH}/TEST*.json"
+  do
```

Comment:
Let's use find instead

```suggestion
  find "${CUCUMBER_REPORTS_PATH}" -name 'TEST*.json' -print0 |
  while IFS= read -r -d '' f
  do
```